monash-human-power/dashboard
# Add stories for Scatter Graph

## Description

`ScatterChart` is now in Typescript and has stories for each of the XDistance charts that are based on `ScatterChart`.

My implementation of the data generation for the stories may be more suited to be exported and used for other data generation uses in other stories.

## Screenshots

![image](https://user-images.githubusercontent.com/50545432/95673817-d4cd8b00-0bf7-11eb-8a44-ebc14593fb4e.png)

![image](https://user-images.githubusercontent.com/50545432/95673862-10685500-0bf8-11eb-8753-98f15c6cd6ab.png)

![image](https://user-images.githubusercontent.com/50545432/95673867-14947280-0bf8-11eb-9715-094aa1d3b3ef.png)

![image](https://user-images.githubusercontent.com/50545432/95673872-1a8a5380-0bf8-11eb-817a-f833126fcfef.png)

- There are blank versions for each of the charts

## Steps to Test

`ScatterChart` changes

1. Start client of dashboard

2. It should function the same

Stories

1. Run `yarn run storybook`

2. `ScatterChart` stories should look the same as on the client
